submissions/mustbefail/html-movie-seat-booking/index.html

```diff
@@ -12,208 +12,216 @@
     />
     <link rel="preconnect" href="https://fonts.gstatic.com" />
     <link
-      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,900;1,400&display=swap"
       rel="stylesheet"
     />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
     <main>
-        <section class="booking">
-          <div class="poster desctop">
-            <img src="https://www.themoviedb.org/t/p/original/89rjeJhduF2qT6TRrbJvVGEhBpX.jpg" alt="film poster">
+      <div class="booking">
+        <div class="poster desktop">
+          <img
+            src="https://www.themoviedb.org/t/p/original/89rjeJhduF2qT6TRrbJvVGEhBpX.jpg"
+            alt="film poster"
+          />
+        </div>
+        <article class="movie-info">
+          <h1>The Devil All the Time</h1>
+          <p class="tagline">
+            "Everyone ends up in the same <br />
+            damned place."
+          </p>
+          <div class="info-block">
+            <img
+              src="https://www.themoviedb.org/t/p/original/89rjeJhduF2qT6TRrbJvVGEhBpX.jpg"
+              alt="film poster"
+            />
+            <div>
+              <div class="rating">
+                <div class="rating-badge">74</div>
+                <span>Viewers rating</span>
+              </div>
+              <span class="rate tag">R</span>
+              <time class="tag">11/09/2020 (US)</time>
+              <time class="tag">2h 18m </time>
```

Comment:
This value is movie duration, not a specific time like 20 March 2020 14:20. In this case use `<p>`